src/core_plugins/console/public/src/directives/history.html

```diff
@@ -9,7 +9,7 @@
       ng-mouseenter="history.viewingReq = req"
       ng-mouseleave="history.viewingReq = history.selectedReq"
       ng-dblclick="history.restore(req)"
-      title="{{ req.description }}"
+      aria-label="{{:: 'Request: ' + history.describeReq(req) }}"
```

Comment:
Hm, I think this is correct as-is. Based on my understanding,`aria-labelledby` is used to point to another element on the screen which contains text that labels the element, e.g. an input element which is labeled by a label element. In this case, we technically don't even need `aria-label` because the element already contains text which is self-descriptive. But I thought that the text might lack a little context, so I created an `aria-label` so we can prepend the word "Request", just to make it a little clearer. What do you think of this?